docs/includes/topic_neutron-bigip-command-mapping.rst

```diff
@@ -14,46 +14,48 @@ When you issue ``neutron lbaas`` commands on your OpenStack Neutron controller o
-                                           | - BIG-IP advertises that vtep's IP address
-                                           | - the selfip IP address is discoverd by Neutron as the agent's ``tunneling_ip``
-                                           | - new port for the vtep added to the OVS switch
-                                           | - tunnel profiles created on BIG-IP for all tunnel types
-                                           | (not limited to the ``advertised tunnel type`` setting in the config file)
+``systemctl start f5-openstack agent``     | 1. agent reads vtep interface defined in config file
+                                           | 2. BIG-IP advertises that vtep's IP address
+                                           | 3. the selfip IP address is discovered by Neutron as the agent's ``tunneling_ip``
+                                           | 4. new port for the vtep added to the OVS switch
+                                           | 5. tunnel profiles created on BIG-IP for all tunnel types
+                                           | (not just for the ``advertised tunnel type`` setting in the config file) [#]_
 --------------------------------------     ---------------------------------------------------------------------------------
-``neutron lbaas-loadbalancer-create``      | - new partition created using the prefix [#]_ and tenant ID [#]_
-                                           | - fdb records added for all peers in network
-                                           | - new route domain created; traffic group is inherited by default
-                                           | - new self-ip created on specified subnet where the BIG-IP can receive traffic
-                                           | - new tunnel created with vtep as local address (using vxlan profile created
-                                           | when agent started)
-                                           | - snat pool list / snat translation list created on BIG-IP (number of snat
+``neutron lbaas-loadbalancer-create``      | 1. new partition created using the prefix [#]_ and tenant ID [#]_
+                                           | 2. fdb records added for all peers in network
+                                           | 3. new route domain created; traffic group is inherited
+                                           | 4. new self-ip created on specified subnet where the BIG-IP can receive traffic
+                                           | 5. new tunnel created with vtep as local address (using vxlan profile created
+                                           | when agent started) [#]_
+                                           | 6. snat pool list / snat translation list created on BIG-IP (number of snat
                                            | addresses created is defined in agent config file) [#]_
-                                           | - neutron port created for each snat address
```

Comment:
if snat mode is turned off / snat addresses set to 0, the big-ip acts as a gateway so that return traffic from members is routed through it.
if snat mode is turned on / snat addresses set to 0, then auto snat mode is used.